lib/theme/styling/filled_button_style.dart

```diff
@@ -0,0 +1,47 @@
+
+import 'package:flutter/material.dart';
+
+class CustomButtonTheme
+    extends ThemeExtension<CustomButtonTheme> {
+  final ButtonStyle? primaryFilledButtonTheme;
```

Comment:
optionals are not required params, either remove ? or remove required.
prefer to remove "?"

Reply:
Got it